optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/pages/employeeroster/EmployeeRosterViewport.java

```diff
@@ -71,18 +76,21 @@ public void drawGridLinesAt(final IsElement target) {
     }
 
     @Override
-    public void drawTicksAt(final IsElement target) {
-        //FIXME: Make it18n
-        ticks.drawAt(target, this, date -> {
-            final int hours = date.getHour();
-            if (hours == 0) {
-                final String lowerDayOfTheWeek = date.getDayOfWeek().toString().toLowerCase();
-                final String dayOfTheWeek = lowerDayOfTheWeek.substring(0, 1).toUpperCase() + lowerDayOfTheWeek.substring(1);
-                int month = date.getMonth().getValue();
-                return ((month < 10 ? "0" : "") + month) + "/" + date.getDayOfMonth() + " " + dayOfTheWeek.substring(0, 1);
-            } else {
-                return (hours < 10 ? "0" : "") + hours + ":00";
+    public void drawDateTicksAt(IsElement target) {
+        DateTimeFormat dateFormat = DateTimeFormat.getFormat(PredefinedFormat.DATE_FULL);
+        dateTicks.drawAt(target, this, date -> {
+            return dateFormat.format(GwtJavaTimeWorkaroundUtil.toDate(date));
+        });
+    }
+
+    @Override
+    public void drawTimeTicksAt(IsElement target) {
+        DateTimeFormat timeFormat = DateTimeFormat.getFormat(PredefinedFormat.TIME_SHORT);
+        timeTicks.drawAt(target, this, date -> {
+            if (date.getHour() == 0) {
+                return "";
             }
```

Comment:
We might wanna get rid of that check and just display the 00:00 time as well. I think it may look better on the screen.

Reply:
Then it will go through the line (or did I forget to add the day lines)

Reply:
Oh, okay. We might wanna wait for UX's input on that matter then.